backend/Controllers/SightingController.cs

```diff
@@ -27,20 +37,56 @@ public ActionResult<ListResponse<Sighting>> GetSightingsBySpeciesId([FromRoute]
         {
             var sightings = _sightings.GetSightingsBySpeciesId(speciesId);
             return new ListResponse<Sighting>(sightings);
-        }   
+        }
 
-        [HttpGet("/pending")] 
-        public ActionResult<ListResponse<Sighting>>GetPendingSightings()
+        [HttpGet("pending")]
+        public ActionResult<ListResponse<Sighting>> GetPendingSightings()
         {
             var pendingSightings = _sightings.GetPendingSightings();
             return new ListResponse<Sighting>(pendingSightings);
-        }   
+        }
 
         [HttpPost]
         public ActionResult CreateSighting([FromBody] CreateSightingRequest createSightingRequest)
-        {   
+        {
             var createdSighting = _sightings.CreateSighting(createSightingRequest);
             return Created("/api", createdSighting);
         }
+
+        [HttpPatch("{sightingId}/confirmation")]
+        public ActionResult ChangeConfirmationStatus(
+            [FromHeader] string authorization, 
+            [FromRoute] int sightingId,
+            [FromBody] ConfirmOrRejectRequest confirmOrRejectRequest)
+        {
+            if (authorization is null)
+            {
+                return new UnauthorizedResult();
+            }
+            try
+            {
+                (string, string) usernamePassword = AuthHelper.GetUsernameAndPassword(authorization);
+                string username = usernamePassword.Item1;
+                string password = usernamePassword.Item2;
```

Comment:
As above, destructuring:
```suggestion
                (string username, string password) = AuthHelper.GetUsernameAndPassword(authorization);
```